src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka_gfx.c

```diff
@@ -0,0 +1,209 @@
+    0x20C54207, 0x524B6A89, 0x62496287, 0x5A4749C7, 0x31853945, 0x41C55185, 0x49C53A45, 0x4A495209, 0x62475A05,
+    0x52076207, 0x52055A47, 0x4A0949C5, 0x52095247, 0x52075249, 0x5A494A87, 0x42473A09, 0x190539C9, 0x5A095A0B,
+    0x5A075A07, 0x4A0549C7, 0x41453985, 0x41C94187, 0x420941C5, 0x4A054A09, 0x49C55205, 0x4A054A07, 0x5A0749C3,
+    0x49C55205, 0x520551C7, 0x59C75207, 0x4A474A07, 0x3A073A07, 0x190339C5, 0x52455A49, 0x5A075249, 0x52074A07,
+    0x41C34985, 0x4A055A47, 0x49C74A09, 0x420951C7, 0x49C75207, 0x49C551C5, 0x51875207, 0x52455A45, 0x52075209,
+    0x59C74A05, 0x498549C5, 0x3A074207, 0x20C53985, 0x52475209, 0x52055247, 0x62096247, 0x39874207, 0x4A075245,
+    0x52474249, 0x42474A09, 0x4A055207, 0x41C541C5, 0x41C55A45, 0x52075209, 0x62895A07, 0x52875203, 0x49C539C7,
+    0x3A053A07, 0x19033A07, 0x51876209, 0x5A476207, 0x5A875A09, 0x41874147, 0x49C749C5, 0x49C53A07, 0x49C94205,
+    0x498541C5, 0x41474185, 0x41C54A07, 0x4A0551C7, 0x498549C5, 0x4A074207, 0x39473985, 0x318739C7, 0x190341C5,
+    0x5A476205, 0x624962C9, 0x62475A47, 0x49C74985, 0x41C74985, 0x42054207, 0x41C741C9, 0x39C74149, 0x41474147,
+    0x39874987, 0x41874987, 0x41C749C5, 0x41854987, 0x49893985, 0x29C739C5, 0x21074185, 0x62456249, 0x624B6A89,
+    0x62496A89, 0x52475249, 0x52474A87, 0x4A474A07, 0x524B5247, 0x4A474A09, 0x4A074987, 0x4A095A09, 0x5A896249,
+    0x62076205, 0x6247624B, 0x528951C9, 0x41C54205, 0x11054207, 0x52496209, 0x52496287, 0x62476A87, 0x5A895A87,
+    0x6247520B, 0x4A094AC9, 0x5A895A07, 0x5A0B5A47, 0x52074A47, 0x5A896289, 0x62876A89, 0x6A476AC9, 0x62475209,
+    0x5A474A09, 0x4A094A07, 0x194341C9, 0x62475A47, 0x49CB3945, 0x5A076287, 0x6A496289, 0x62895A87, 0x5A895249,
+    0x520951C9, 0x310749C7, 0x51C75A49, 0x62476289, 0x620962C7, 0x62875A47, 0x4A0B2985, 0x4A4B5247, 0x41C54A09,
+    0x19454209, 0x5A8972CD, 0x9C5B4A0B, 0x62476AC9, 0x62875AC9, 0x62095A0B, 0x524B524B, 0x6B0F9419, 0x418B41C7,
+    0x4A075A07, 0x62496287, 0x62876AC7, 0x628972CF, 0x8C5D4A4B, 0x52874A47, 0x424B4A07, 0x18C54985, 0x5A896287,
+    0x7B4D6247, 0x6247728B, 0x62C76289, 0x6A075ACB, 0x4A474A47, 0x5A876AD1, 0x52455205, 0x52095A07, 0x62876A89,
+    0x62496247, 0x6A496249, 0x7B0B5209, 0x52874A49, 0x42495209, 0x19434207, 0x52876207, 0x52495A49, 0x62095A09,
+    0x5A095289, 0x62475207, 0x4A874A89, 0x52495287, 0x52495205, 0x42075A05, 0x62476247, 0x5A095249, 0x52496249,
+    0x62495249, 0x5A874A07, 0x49C74207, 0x19453945, 0x41C739C7, 0x420741C9, 0x41C74209, 0x42054187, 0x3A0541C5,
+    0x3A0931C7, 0x39C541C7, 0x49C73147, 0x3A0541C5, 0x49C549C7, 0x398941C7, 0x420749C7, 0x49C739C7, 0x420731C7,
+    0x320741C7, 0x08C51903, 0x19431143, 0x21451903, 0x21432105, 0x19432105, 0x19051905, 0x11031905, 0x21451907,
+    0x11072105, 0x19031943, 0x190518C5, 0x214518C5, 0x21432145, 0x11032103, 0x19051905, 0x19031903,
+};
+
+// probably unused?
+static UNK_TYPE D_808772E0[] = { 0x00000000, 0x00000000 };
```

Comment:
Is this needed? I'm thinking it might just be padding.

Reply:
It is needed, but it works either as an array of one zero or two zeros. Removing it completely unfortunately doesn't match.

Reply:
By any chance, does it match if the texture above is an array of `u64`?

Reply:
Nope, doesn't seem to change a thing.